client-ts/signalr/src/HubConnection.ts

```diff
@@ -267,6 +267,9 @@ export class HubConnection {
         const removeIdx = handlers.indexOf(method);
         if (removeIdx !== -1) {
             handlers.splice(removeIdx, 1);
+            if(handlers.length === 0) {
```

Comment:
nit: add a space between the if and the (

Reply:
(Note to self: Stop doing PR's on the phone)